### PR TITLE
Update explore-intro.html for Windows system

### PR DESCRIPTION
**Pull request by FreezeIce1**
Some operation in Windows system cannot execute, so add some operation for Windows system learner.

### CODE REVIEW

File: content/en/docs/tutorials/kubernetes-basics/explore/explore-intro.html [hunk 1]
```diff
@@ -153,8 +153,13 @@ <h3>Show the app in the terminal</h3>
                 To get the Pod name and store it in the <tt>POD_NAME</tt> environment variable:</p>
                 <p><code><b>export POD_NAME="$(kubectl get pods -o go-template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')"</b></code><br />
                 <code><b>echo Name of the Pod: $POD_NAME</b></code></p>
+                <p>In Windows system could use:</p>
```
Comment (windsonsea) [inline content/en/docs/tutorials/kubernetes-basics/explore/explore-intro.html:156]: It's recommended to use bullets to describe different cases like:

- macOS
- Windows
- Other

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-44016--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/explore/explore-intro/

**Comment by kbhawkey** [comment]
👋 @FreezeIce1 .
There are several sections in this tutorial where the shell commands could be customized for the operating system.
Instead, would it help to add a note that the commands displayed in the tutorial are intended to be run on Linux platforms?

**Comment by natalisucks** [comment]
@FreezeIce1 Hi there! Do you have bandwidth to address Karen and Michael's feedback? Thanks!

**Comment by FreezeIce1** [comment]
> 👋 @FreezeIce1 . There are several sections in this tutorial where the shell commands could be customized for the operating system. Instead, would it help to add a note that the commands displayed in the tutorial are intended to be run on Linux platforms?

Yes, I think so. For a new learner, maybe he/she is the first time to know about k8s. We don't know they use which desktop. If a learner always face problem with environment, I think it is easy to give up.

**Pull request closed without merging**